GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3246,13 +3253,22 @@ private void HandleDrop(DragEventArgs e)
                 return;
             }
 
-            foreach (string path in paths)
+            foreach (string path in paths.Where(IsFileExistingInRepo))
```

Comment:
Are you sure this is needed ? fileTree.SelectFileOrFolder already do a ``filePath.IndexOf(Module.WorkingDir) != 0``

Reply:
Avoid LINQ where possible, it has perf overhead. We're already traversing the collection, so just check in each iteration.

Reply:
@bansan85 yes, it is needed to filter before trying to select it otherwise there are case where you will select the tabs even of the content dropped has nothing to do with file paths belonging to this repository (something you complain in your issue...)

Reply:
@RussKie I will remove the use of LINQ...